contributing.md

```diff
@@ -29,3 +12,26 @@ Before edited code can be used on the master branche, a pull request needs to be
 
 #### -issues
 When a (preceived) problem is detected in the code, an issue can be made. This issue will document the problem and make it visible to the team.
+
+# Testing
+## [Mocha:](https://mochajs.org/)
+Mocha is a test executor. This package is in control of all the tests that are to be executed on the code. This framework works with plugins like Sinon, Chai and Sonar that provide the actual tools to test the code, and Mocha makes it possible to use "npm test" in the terminal and collectively execute the tests.
+
+## [Chai:](https://www.chaijs.com/)
+Chai is a framework that makes test conditions easy. Chai makes it possible to make expect, should and assert statements. When one of the expects fails, it can be assumed that there was undefined behaviour.
+
+## [Sinon:](https://sinonjs.org/)
+To make the code reliable, multiple scenario's need to be tested. Sinon helps with that. It makes mock-databases, mock-entities, stubs and spies so this can be tested in a development enviroment, without using external tools like databases and/or filling databases with test data. This will ensure that all the scenario's that can be thought of in development will be tested if you run a test after making edits to the code.
+
+## [Travis:](https://travis-ci.org/)
+Travis is a continuous integration platform. This program makes distributing changes in the code to working servers easier. When changes are made and are accepted in Github, the changes are automatically rolled out to the servers that run on the software.
```

Comment:
The thing you're describing here is `continuous deployment` which doesn't really apply to us yet, as there are no servers. We use Travis to test, perform code analysis. In the future we might produce artifacts with it.